app/models/driver.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class Driver < ApplicationRecord
+  # after_initialize :set_availability
+  has_many :trips
+  validates :name, presence: true
+  validates :vin, presence: true
+
+  def average_rating
+    ratings = trips.map {|trip| trip.rating}
+    if ratings = 0
+      return "NIL"
+    end
```

Comment:
Because you used `=` instead of a `==` this condition is always true.  

Also, you probably wanted to say `ratings.empty?` instead:
```suggestion
    if ratings.empty?
      return "NIL"
    end
```